Rakefile

```diff
@@ -174,8 +174,8 @@ namespace :spec do
       PATH='#{TEST_GEMS_DIR}/bin:#{path}' \
       BUNDLE_GEMFILE='#{PUPPET_SRC}/Gemfile' \
       GEM_HOME='#{TEST_GEMS_DIR}' GEM_PATH='#{TEST_GEMS_DIR}' \
-      lein run -m org.jruby.Main \
-        -S bundle install --without extra development --path='#{TEST_BUNDLE_DIR}' --retry=3
+      lein #{profile} run -m org.jruby.Main \
```

Comment:
I don't think this exists in master? I'll double check in a second.

Reply:
Didn't get this far, looks like you and travis are seeing the same thing I am. However, the reason I thought this line was off is because I *think* we used to use this for switching between 9k and 1.7 in our specs.

Reply:
Ah okay, makes sense. I'll try to fix the packaging issue and remove this.